server/pulp/plugins/loader/manager.py

```diff
@@ -1,27 +1,13 @@
-# -*- coding: utf-8 -*-
-#
-# Copyright © 2011-2012 Red Hat, Inc.
-#
-# This software is licensed to you under the GNU General Public License as
-# published by the Free Software Foundation; either version 2 of the License
-# (GPLv2) or (at your option) any later version.
-# There is NO WARRANTY for this software, express or implied, including the
-# implied warranties of MERCHANTABILITY, NON-INFRINGEMENT, or FITNESS FOR A
-# PARTICULAR PURPOSE.
-# You should have received a copy of GPLv2 along with this software; if not,
-# see http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt
-
-import copy
-import logging
 from gettext import gettext as _
 from pprint import pformat
+import copy
```

Comment:
I would prefer this to be alphabetical.

https://github.com/pulp/pulp_docker/pull/52#discussion_r22557739